docs/onecodex.lib.rst

```diff
@@ -0,0 +1,53 @@
+onecodex.lib package
+====================
+
+Submodules
+----------
+
+onecodex.lib.auth module
+------------------------
+
+.. automodule:: onecodex.lib.auth
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+onecodex.lib.download module
+----------------------------
+
+.. automodule:: onecodex.lib.download
+   :members:
+   :undoc-members:
+   :show-inheritance:
```

Comment:
I think we can remove all the `:show-inheritance`s from the files because it's configured by default in `conf.py`

Can we auto-configure `:undoc-members` and `:members` too?

Reply:
I actually ended up disabling this for everything but `SampleCollection` (it's nice to have it there because of all of the mixins).